gtfs-validator/main/src/test/java/org/mobilitydata/gtfsvalidator/validator/TripTravelSpeedValidatorTest.java

```diff
@@ -0,0 +1,111 @@
+                                             .setStopId("stopA")
+                                             .setStopSequence(1)
+                                             .setDepartureTime(GtfsTime.fromString("12:20:30"))
+                                             .build();
+  private final GtfsStopTime stopTime2 = new GtfsStopTime.Builder()
+                                             .setCsvRowNumber(6)
+                                             .setTripId("trip1")
+                                             .setStopId("stopB")
+                                             .setStopSequence(2)
+                                             .setArrivalTime(GtfsTime.fromString("12:20:31"))
+                                             .build();
+  private final GtfsStopTime stopTime3 = new GtfsStopTime.Builder()
+                                             .setCsvRowNumber(6)
+                                             .setTripId("trip1")
+                                             .setStopId("stopB")
+                                             .setStopSequence(2)
+                                             .setArrivalTime(GtfsTime.fromString("23:20:31"))
+                                             .build();
+  private final GtfsStop stop1 = new GtfsStop.Builder()
+                                     .setCsvRowNumber(1)
+                                     .setStopId("stopA")
+                                     .setStopLat(28.06d)
+                                     .setStopLon(-82.416d)
+                                     .build();
+  private final GtfsStop stop2 = new GtfsStop.Builder()
+                                     .setCsvRowNumber(2)
+                                     .setStopId("stopB")
+                                     .setStopLat(28.08d)
+                                     .setStopLon(-82.416d)
+                                     .build();
+
+  private final NoticeContainer noticeContainer = new NoticeContainer();
+  private final TripTravelSpeedValidator validator = new TripTravelSpeedValidator();                                   
+  
+  @Test
+  public void fastTravelGenerateNotice() {
+    validator.tripTable = GtfsTripTableContainer.forEntities(Arrays.asList(trip1), noticeContainer);
+
+    validator.stopTimeTable = GtfsStopTimeTableContainer.forEntities(
+        Arrays.asList(stopTime1, stopTime2), noticeContainer);
+
+    validator.stopTable =
+        GtfsStopTableContainer.forEntities(Arrays.asList(stop1, stop2), noticeContainer);
+
+    validator.validate(noticeContainer);
+    
+    assertThat(noticeContainer.getNotices().size()).isEqualTo(1);
+    Map<String, Object> noticeInfo = noticeContainer.getNotices().get(0).getContext();
+    assertThat(noticeInfo.get("tripId")).isEqualTo("trip1");
+    // To compare doubles need to consider precision
+    assertThat((double)noticeInfo.get("speedkmh")).isWithin(1.0e-10).of(8006.045528786268);
```

Comment:
perhaps 1.0e-10 should be named 'EPSILON'?

Reply:
Sure done. Definitely makes it more readable.